tools/scripts/gen-env-files.sh

```diff
@@ -493,3 +496,23 @@ Auth__Keycloak__Secret={YOU WILL NEED TO GET THIS FROM KEYCLOAK}
 Service__OutputPath=../data/image" >> ./services/net/image/.env
     echo "./services/net/image/.env created"
 fi
+
+## Transcription service
+if test -f "./services/net/transcription/.env"; then
+    echo "./services/net/transcription/.env exists"
+else
+echo \
+"ASPNETCORE_ENVIRONMENT=Development
+ASPNETCORE_URLS=http://+:8081
+
+Auth__Keycloak__Authority=http://host.docker.internal:40001/auth/realms/tno
+Auth__Keycloak__Audience=tno-service-account
+Auth__Keycloak__Secret={YOU WILL NEED TO GET THIS FROM KEYCLOAK}
+
+Service__ApiUrl=http://host.docker.internal:40010/api
```

Comment:
The port should be a variable I believe.

Reply:
Fixed.